@shared/api/internal.ts

```diff
@@ -455,7 +455,6 @@ export const getAccountIndexerBalances = async (
   if ("error" in data && (data?.error?.horizon || data?.error?.soroban)) {
     const _err = JSON.stringify(data.error);
     captureException(`Failed to fetch account balances - ${_err}`);
-    throw new Error(_err);
```

Comment:
throw an error here leads to the redux thunk rejecting - which just leads to the generic `Failed to fetch account balances error`. If we don't throw, we can show the user a Horizon specific error